### ISSUE TITLE
what is IO

### ISSUE THREAD

**Issue by EvenZ97** [limbo018/DREAMPlace#154]
Dear Dr.Lin, I hope this message finds you well. I have read the paper "DREAMPlace “ and "DREAMPlace 2.0", I found that the table of the paper results has the information about IO, and its unit is time. I don't really know what it exactly means , I think it maybe means the runtime of file reading or writting, or maybe the runtime of the pins placement , and I didn't find it in the output of the results. Thank you for your time and assistance. I appreciate your guidance in resolving these issues.

**Comment by Constwelve** [comment]
Yes, in the paper IO means the runtime of file reading and writting.
